crates/api_common/src/utils.rs

```diff
@@ -158,6 +152,26 @@ pub async fn mark_post_as_read(
   Ok(())
 }
 
+/// Updates the read comment count for a post. Usually done when reading or creating a new comment.
+#[tracing::instrument(skip_all)]
+pub async fn update_read_comments(
+  person_id: PersonId,
+  post_id: PostId,
+  read_comments: i64,
+  pool: &mut DbPool<'_>,
+) -> LemmyResult<()> {
+  let person_post_agg_form = PersonPostAggregatesForm {
+    person_id,
+    post_id,
+    read_comments,
+    ..PersonPostAggregatesForm::default()
+  };
+  PersonPostAggregates::upsert(pool, &person_post_agg_form)
+    .await
+    .with_lemmy_type(LemmyErrorType::CouldntFindPost)?;
+  Ok(())
```

Comment:
Nitpick: I think adding newlines between lines 168 and 169, and between 171 and 172, would make this easier to read.

Reply:
Can do.